commonui/src/test/java/com/hyperwallet/android/ui/common/util/TransferMethodUtilsTest.java

```diff
@@ -63,7 +64,7 @@ public void setup() {
     public void testGetTransferMethodName_returnsCorrectNameForName() {
         when(mContext.getString(R.string.bank_card)).thenReturn("My bank card");
         String transferMethodName = TransferMethodUtils.getTransferMethodName(mContext, BANK_CARD);
-        assertThat(transferMethodName, is("My bank card"));
+        MatcherAssert.assertThat(transferMethodName, Matchers.is("My bank card"));
```

Comment:
please import the whole static class for `MatcherAssert` and `Matchers` please

Reply:
removed